vault/dendron.topic.links.md

```diff
@@ -1,97 +1,80 @@
 ---
 id: 3472226a-ff3c-432d-bf5d-10926f39f6c2
 title: Links
-desc: 'Dendron supports many ways of linking notes and files'
-updated: 1658768400279
+desc: Create links to various locations using basic and extended syntax
+updated: 1659082798415
 created: 1595003088839
+config:
+  global:
+    enableChildLinks: false
 ---
 
 ## Summary
 
 {{fm.desc}}.
 
-## Types
-- Markdown links (can link to any file):
-    - `[link text](dendron.note.name.md)` -> [link text](dendron.note.name.md)
-    - `[link text](./path/to/filename.md)` -> [link text](./path/to/filename.md)
-    - `[link text](assets/filename.pdf)` -> [link text](assets/filename.pdf)
-- Wiki links: `[[dendron.note.name]]` -> [[dendron.note.name]]
-- Image links: `![image alt text](http://example.com/or/path/to/image)`
-
-## Concepts
-
-### Note Files
-
-A note file is a Dendron Markdown note.
-
-### Regular Files
-
-Regular files are any assets that are not Dendron Markdown notes. 
+Dendron provides many ways to create links between notes and assets. This range from regular markdown links to extended syntax such as wikilinks, file links, and much more.
 
 ## Features
 
-### Wiki Links
+### Link to various locations with both regular Markdown links and extended link syntax
 
-![[dendron://dendron.dendron-site/dendron.topic.links.wiki-link]]
-
-### File Links
+- Dendron is built on top of Markdown, and regular Markdown links work out of the box. 
+- Furthermore, Dendron extends the syntax of regular Markdown links and support Wikilinks. Many convenient features are built around wikilinks to support various use cases when creating links between notes in your knowledge base.
 
-![[dendron://dendron.dendron-site/tags.stage.sprout]]
+### Manipulate links using various commands
```

Comment:
this is too vague. what sort of manipulation? instead of throwing them in the same category, break them out (eg. see above example: `Quick Link Creation`)